support/pullsources/sources_list.sha1

```diff
@@ -219,6 +223,7 @@ tcl8.6.1-src.tar.gz - 5c83d44152cc0496cc0847a2495f659502a30e40
 tcpdump-4.7.3.tar.gz - 4f085cef7cd4aedc9e402021ec11e3a8b23a6926
 tcsh-6.18.01.tar.gz - eee2035645737197ff8059c84933a75d23cd76f9
 tdnf-1.0.1.tar.gz - bff1ffdce4f7c319c06da3a76f8be70f76be1246
+tdnf-1.0.tar.gz - f2e10a64fd564082c64eb4e4b585d7192cf47ff5
```

Comment:
Please make sure you are not adding any extra file which you did not intended to be added. TDNF was added and removed by Priyesh so make sure you do not upload it again.